EpiserverRedirects/Model/RedirectRule/RedirectRule.cs

```diff
@@ -127,6 +138,10 @@ public void FromManual()
         public string CreatedBy { get; set; }
         public string Notes { get; set; }
         public int Priority { get; set; }
+
+        private string _oldPattern;
```

Comment:
it's nice to keep those private matches next to its public pair (_oldPattern next to OldPattern)